### ISSUE TITLE
{ 'pmin': 600, 'pmax': 3600 }

### ISSUE THREAD

**Issue by MLopezJ** [MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#7]
# TL;DR
Value is originated because object was defined as "Telemetry" on Coiote Dashboard. If it is defined as "Property" it does not appear. 

It is required to understand the implications of it.

# Issue Discussion
Value with new struct is been received in [input data](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#expected-input).

its follow the next struct:

``` { 'pmin': 600, 'pmax': 3600 }```

And is present on:

- `Object` 6 `Prop` 0
- `Object` 6 `Prop` 1
- `Object` 3303 `Prop` 5700
- `Object` 3304 `Prop` 5700
- `Object` 3315 `Prop` 5700

## Questions

~Is this struct defined on `LwM2M` ?~

~Is this new value `valid` as a result from Azure integration or it is something that is not expected ?~

**Comment by MLopezJ** [comment]
Conclusion of this issue could modify steps described in #6

**Comment by MLopezJ** [comment]
> Is this struct defined on `LwM2M` ?

No. All expected values from those props should be **float** type

> * `Object` 6 `Prop` 0
> * `Object` 6 `Prop` 1
> * `Object` 3304:1.1 `Prop` 5700
> * `Object` 3315 `Prop` 5700

| ID  |  Object  | Prop | Name | Expects | LwM2M Def |
|---|---|---|---|---|---|
|  6 |  Location | 0 | Latitude | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/6.xml) | 
|  6 |  Location | 1 | Longitude | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/6.xml) | 
| 3303  | Temperature  | 5700  | Sensor Value | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/3303.xml)  | 
| 3304  | Humidity  | 5700  | Sensor Value | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/3304.xml)  | 
| 3315  | Barometer | 5700 | Sensor Value | **float** |   [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/3315.xml) |

**Comment by MLopezJ** [comment]
## `Location` in `Coiote`

![Screenshot from 2023-02-09 17-05-42](https://user-images.githubusercontent.com/8351865/217868470-9c6aaeea-e68d-469b-87b7-d2ef1b36ac76.png)

Values of prop `0` and `1` are not provided.

## `Humidity` in `Coiote`

![Screenshot from 2023-02-09 17-06-18](https://user-images.githubusercontent.com/8351865/217868682-4d8b0e4a-e70f-480a-bbc3-7d8480dc3f62.png)

Value of prop `5700` is `12.713`

## `Barometer` in `Coiote`

![Screenshot from 2023-02-09 17-06-42](https://user-images.githubusercontent.com/8351865/217868644-24892fff-1cb5-4220-a115-a73411474563.png)

Value of prop `5700` is `99377`

**Comment by MLopezJ** [comment]
This mean there should be an error or a miss configuration of the source where the expected input was taken. 

Discussion about it [here](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/8)

**Comment by MLopezJ** [comment]
> Is this new value `valid` as a result from Azure integration or it is something that is not expected ?

There is not know if it value is valid or nor, but what it is known is that value was returned because it was defined as `telemetry` data. When objects were defined as `property` data, result was the expected, as it is described [here](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/8#issuecomment-1428210012)

## Questions

What is `telemetry` data?
What is `propery` data?
How Coiote interpreted `telemetry` data ?

**Comment by MLopezJ** [comment]
Close issue because remaining questions are not a priority right now. 

Issue will be re-open if discussion rewarding this recover importance

**Comment by MLopezJ** [comment]
> ## Questions
> What is `telemetry` data? 
> What is `propery` data? 
> How Coiote interpreted `telemetry` data ?

The solution relies on the assumption that setting all the data as "property" data would give us the expected input. However, it is important to understand which implications have that change and that's the reason why the priority of getting the answer to the last questions have change. 

Discussion will be addressed here: [Telemetry vs Property data  ](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/18)
